documentation/player-portal/android/player-config-android.md

```diff
@@ -61,7 +87,7 @@ The PKMediaEntry object contains a list of "PKMediaSources". All sources relate
 
 To learn more, see [PKMediaSource](https://github.com/kaltura/playkit-android/blob/develop/playkit/src/main/java/com/kaltura/playkit/PKMediaSource.java)
 
-#### Manually a Create Media Source  
+#### Manually Create Media Source   
```

Comment:
Should read: "Manually create a media source"

Reply:
fixed